### ISSUE TITLE
Firefox: Listbox is not working properly with NVDA

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9365]
### Steps to reproduce:
1. Open NVDA
2. Open this link in Firefox
https://sites.google.com/site/secretariapiloto188cvv/diarioplantao
3. Type your e-mail address and press "next"
4. Right to the buttons "back" and "next" there is a listbox. Navigate in browse mode to it and press enter
5. Press alt+down arrow or arrow keys to select an item
6. Press tab and shift+tab to land on it again in focus mode
7. Press alt+down arrow and select an item

### Actual behavior:
In browse mode, when pressing enter on the listbox, NVDA activates focus mode but you can not interact with the items in the list. After pressing tab and shift+tab you land again on the list and then you can select items. But there seems to be some offset errors.

### Expected behavior:
In browse mode, When pressing enter on the listbox NVDA should alow you to expand the listbox with alt+down arrow like in other lists and there should not appear any selection fails in the log file.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2019.1 Beta 1

#### Windows version:
Windows 10 1809 Update

#### Name and version of other software in use when reproducing the issue:
Firefox 65.0.2

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
yes, 2018.3.2 has the same issue.

**Comment by Adriani90** [comment]
When selecting an item in the listbox, I get following fails in the log file:

```
IO - speech.speak (12:24:59.936):
Speaking [u'Bauru  nicht ausgew\xe4hlt  5 von 39']
IO - inputCore.InputManager.executeGesture (12:25:00.198):
Input: kb(laptop):downArrow
IO - inputCore.logTimeSinceInput (12:25:00.338):
0.139 sec since input
IO - speech.speak (12:25:00.338):
Speaking [u'Bel\xe9m  nicht ausgew\xe4hlt  6 von 39']
IO - inputCore.InputManager.executeGesture (12:25:01.644):
Input: kb(laptop):enter
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.871):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.885):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.907):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.926):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.944):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.967):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:01.989):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:02.009):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:02.020):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:02.032):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - RPC process 15296 (firefox.exe) (12:25:02.036):
Thread 520, build\x86_64\vbufBackends\gecko_ia2\gecko_ia2.cpp, GeckoVBufBackend_t::getSelectedItem, 356:
accSelection failed with -2147023113

DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (12:25:02.045):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))
DEBUGWARNING - RPC process 15296 (firefox.exe) (12:25:02.176):
Thread 520, build\x86_64\vbufBackends\gecko_ia2\gecko_ia2.cpp, GeckoVBufBackend_t::getSelectedItem, 356:
accSelection failed with -2147023113
```

**Comment by Adriani90** [comment]
cc: @jcsteh, @leonardder

**Comment by jcsteh** [comment]
This should be fixed in Firefox 66, which has now been released. Please update and re-test.

**Comment by Adriani90** [comment]
Testing with Firefox 66.0.1, it works as expected. I can interact with the elements and there are no errors in the log. Thanks for looking into this. Closing as works for me.
